Dockerfile-desktop-linux

```diff
@@ -1,12 +1,6 @@
 FROM --platform=linux/amd64 golang:1.21.7-bullseye@sha256:447afe790df28e0bc19d782a9f776a105ce3b8417cdd21f33affc4ed6d38f9d5
 LABEL maintainer="Fleet Developers"
 
-RUN apt-get update && apt-get install -y \
-	gcc \
-	libgtk-3-dev \
-	libayatana-appindicator3-dev \
-	&& rm -rf /var/lib/apt/lists/*
```

Comment:
This is not needed anymore because we've recently replaced https://github.com/getlantern/systray with a fork https://github.com/fyne-io/systray (which doesn't have any GTK dependencies on Linux).